pymove/utils/_winmock.py

```diff
@@ -0,0 +1,15 @@
+class WindowsUser:
+  "Mock class for module errors on windows"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [First line should end with a period, question mark, or exclamation point (not 's') (D415)](https://app.codacy.com/gh/InsightLab/PyMove/pullRequest?prid=5497268)